### PR TITLE
3.1 Harvest Visualization and Patching Documentation

### PR DESCRIPTION
**Pull request by obrienben**
Changes
- 3.1 upgrade guide
- Removed references to H1 and bandwidth
- Two new sections in the User Manual
  - User Manual -> Target Instance Quality Review -> Harvest Analysis
  - User Manual -> Target Instance Quality Review -> Patching
- Removed redundant tree view section

To build the docs, instructions are in the webcurator-docs readme

### CODE REVIEW

File: webcurator-docs/guides/user-manual.rst [hunk 1]
```diff
@@ -1982,93 +1974,540 @@ Target Instance view page corresponding to that particular harvest which
...
-   multiple prune and import operations before saving - as a new Harvest
-   Result is created after each operation which can be a very resource
-   intensive operation on the server.
+  The columns available in the Patch Harvest are:
 
-   |image55|
+  - *Option* - Patching action (PRUNE or RECRAWL)
+  - *Target* - URL to be patched
+  - *File* - local file to be imported
+  - *Modified Mode* - //TODO
```
Comment (hannakoppelaar) [inline webcurator-docs/guides/user-manual.rst:2434]: Same TODO as before

**Review by hannakoppelaar — COMMENTED** [review]
There's one TODO issue concerning the semantics of the ModifiedMode field in the bulk import spreadsheet (user-manual.rst)

File: webcurator-docs/guides/user-manual.rst (hunk 1 above)
Comment (obrienben) [reply webcurator-docs/guides/user-manual.rst]: Added Modified Mode description

File: webcurator-docs/guides/user-manual.rst [hunk 2]
```diff
@@ -1982,93 +1974,540 @@ Target Instance view page corresponding to that particular harvest which
...
+
+  - **Option** - specify the action to be taken, either *PRUNE* or *RECRAWL*.
 
--  **Total Success** - The number of documents successfully downloaded from
-   "within" this site or folder.
+  - **LocalFileName** - specify the local path to the file to be imported.
 
--  **Total Failed** - The number of documents unsuccessfully downloaded
-   from "within" this site or folder.
+  - **ModifiedMode** - specify the //TODO
```
[the remark this answers is not in the document]
Comment (obrienben) [reply webcurator-docs/guides/user-manual.rst]: This block was actually redundant so I've removed

**Comment by obrienben** [comment]
@hannakoppelaar have fixed the TODOs

**Comment by hannakoppelaar** [comment]
@leefrank9527 if you're happy with it, I will merge this PR

**Comment by leefrank9527** [comment]
@hannakoppelaar Hi Hanna, I'm reading and testing it. Will finish that soon.

**Comment by leefrank9527** [comment]
@obrienben Got some comments for the doc:
1. The description below "Figure 27. Linked domain nodes": Star nodes are a single domain of URLs that also contain the primary seed of the crawl. In fact, both primary seed contained nodes and secondary seed contained nodes will be drew as star bodes.
2. Can add some description about the right click menu of the domain analysis table below "Figure 30. Domain Analysis", similar to "Figure 29. Node right-click menu".
3. Above "Figure 36. Filtering and search options", the filtering is available with the Path View as well. But it seems there is a performance problem for a big Path View for a broad filter.
4. "To reset the view and clear any fitlered or search results, click the reset button image108."" The icon image is not the latest one.
5. "Figure 52. A running patch in harvest results", "Figure 53. Patch Summary screen": Are they located in the opposite places?

**Comment by obrienben** [comment]
Thanks @leefrank9527. I have made the relevant updates:

1. No change - there is already further description below Figure 27 detailing the star node and secondary seeds.
2. Updated to mention right-click menu for domain analysis table
3. Updated to mention that filtering is available in Crawler Path view also
4. Updated screenshots with latest icon
5. No change - to me the current descriptions match the screenshots

@hannakoppelaar  ready for you to merge

**Pull request merged**